python-bareos/bareos/bsock/lowlevel.py

```
@@ -285,19 +288,28 @@ def __connect_tls_psk(self):
-                    psk=(password, identity),
-                    server_side=False,
-                )
-            except ssl.SSLError as e:
-                # raise ConnectionError(
-                #     "failed to connect to host {0}, port {1}: {2}".format(self.address, self.port, str(e)))
-                # Using a general raise to keep more information about the type of error.
-                raise
+            if "sslpsk3" in sys.modules:
+                context = sslpsk3.SSLPSKContext(ssl.PROTOCOL_TLS_CLIENT)
+                # TLS1.3 only works using Python >= 3.13
+                context.maximum_version = TLSVersion.TLSv1_2
```

Ruff (0.15.7)

[error] Undefined name `TLSVersion`

(F821)